cvsclone.l

```diff
@@ -16,7 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
- * 
+ *
```

Comment:
Aye, you're right. I'll bin the current revision and resubmit without the line ending changes